include/ofi_util.h

```diff
@@ -530,8 +532,9 @@ ssize_t ofi_cq_sreadfrom(struct fid_cq *cq_fid, void *buf, size_t count,
 		fi_addr_t *src_addr, const void *cond, int timeout);
 int ofi_cq_signal(struct fid_cq *cq_fid);
 
-int ofi_cq_write_overflow(struct util_cq *cq, void *context, uint64_t flags, size_t len,
-			  void *buf, uint64_t data, uint64_t tag, fi_addr_t src);
+int ofi_cq_write_overflow(struct util_cq *cq, void *context, uint64_t flags,
```

Comment:
Change to write_aux?

Reply:
I kept the name because this is writing an overflow entry, as opposed to an error, both of which go to the auxiliary queue.  Someday there may be a need to write other types of entries to the auxiliary queue.